src/basic/MAT_MAT_SHARED.cpp

```diff
@@ -0,0 +1,81 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
```

Comment:
I just noticed that the copyright dates are old in these files. Please3 change to 2017-21.

Reply:
fixed